crates/cargo-bsp/src/cargo_communication/cargo_types/cargo_command.rs

```diff
@@ -1,9 +1,9 @@
-//! CreateCommand trait implementation for the Compile/Run/TestParams.
-//! The trait allows getting origin id and creating commands regardless if it is the compile,
-//! run or test request.
+//! CreateCommand and CreateUnitGraphCommand traits implementation for the
+//! Compile/Run/Test/RustWorkspaceParams. The trait allows getting origin id and creating
+//! commands regardless if it is the compile, run, test or rust_workspace request.
 //!
 //! There are two types of commands:
-//! - requested: standard `cargo build`, `cargo run` and `cargo test` to compile,
+//! - requested: standard `cargo build`, `cargo run`, `cargo test` and `cargo check` to compile,
```

Comment:
powinno być chyba "to compile, run, test, and ..."

Reply:
Cargo check to jest tak naprawdę cargo build, tylko bez ostatniego kroku, który normalnie się wykonuje. Dlatego nie wiem jak miałabym to nazwać i uznałam, że "compile" może też opisywać cargo check

Reply:
okej